### PR TITLE
docs: update note about accessing tech

### PR DESCRIPTION
**Pull request by thijstriemstra**
and fix typos

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [x] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: docs/guides/player-workflows.md [hunk 1]
```diff
@@ -353,23 +353,21 @@ console.log(myPlayer.poster());
 
 ## Accessing the Tech on the player
 
-The tech on the player can only be accesed by pasing `{IWillNotUseThisInPlugins: true}` into the `tech()`
+The tech on the player can be accessed by passing any argument into the `tech()`
```
Comment (gkatsev) [inline docs/guides/player-workflows.md:356]:
```suggestion
The tech on the player can be via `tech()`. Passing any argument will silence the warning that is logged.
```

**Review by gkatsev — COMMENTED** [review]
Thanks, we completely forgot this one. One thought for rewording the sentence.

**Comment by thijstriemstra** [comment]
committed suggestion, thx.

**Review by gkatsev — APPROVED** [review]
Thanks!

**Pull request merged**